core/src/main/scala/org/apache/spark/Dependency.scala

```diff
@@ -102,6 +102,12 @@ class ShuffleDependency[K: ClassTag, V: ClassTag, C: ClassTag](
    */
   private[spark] var mergerLocs: Seq[BlockManagerId] = Nil
 
+  /**
+   * Stores the information about whether the shuffle merge is finalized for the shuffle map stage
+   * associated with this shuffle dependency
+   */
+  private[this] var shuffleMergedFinalized: Boolean = false
```

Comment:
super nit: `shuffleMergedFinalized` -> `_shuffleMergeFinalized` ?
Also, do we want to move this to `ShuffleDependency` instead ?

Reply:
I think this is with in `ShuffleDependency` class only but `ShuffleDependency` class is in `Dependency.scala` file.

Reply:
My bad, you are right, it is in `ShuffleDependency` :-)